tools/pelican/content/ipynb/tutorial-accessing-cases-and-deaths.ipynb

```diff
@@ -0,0 +1,6219 @@
+       "L 336.827815 193.270198 \n",
+       "L 339.045033 210.45913 \n",
+       "L 341.262252 201.864664 \n",
+       "L 343.47947 206.161897 \n",
+       "L 345.696689 201.864664 \n",
+       "L 350.131126 201.864664 \n",
+       "L 352.348344 210.45913 \n",
+       "L 354.565563 193.270198 \n",
+       "L 356.782781 201.864664 \n",
+       "L 359 214.756364 \n",
+       "L 359 214.756364 \n",
+       "\" style=\"fill:none;stroke:#e24a33;stroke-linecap:square;stroke-width:1.5;\"/>\n",
+       "   </g>\n",
+       "   <g id=\"patch_3\">\n",
+       "    <path d=\"M 24.2 224.64 \n",
+       "L 24.2 7.2 \n",
+       "\" style=\"fill:none;stroke:#ffffff;stroke-linecap:square;stroke-linejoin:miter;\"/>\n",
+       "   </g>\n",
+       "   <g id=\"patch_4\">\n",
+       "    <path d=\"M 359 224.64 \n",
+       "L 359 7.2 \n",
+       "\" style=\"fill:none;stroke:#ffffff;stroke-linecap:square;stroke-linejoin:miter;\"/>\n",
+       "   </g>\n",
+       "   <g id=\"patch_5\">\n",
+       "    <path d=\"M 24.2 224.64 \n",
+       "L 359 224.64 \n",
+       "\" style=\"fill:none;stroke:#ffffff;stroke-linecap:square;stroke-linejoin:miter;\"/>\n",
+       "   </g>\n",
+       "   <g id=\"patch_6\">\n",
+       "    <path d=\"M 24.2 7.2 \n",
+       "L 359 7.2 \n",
+       "\" style=\"fill:none;stroke:#ffffff;stroke-linecap:square;stroke-linejoin:miter;\"/>\n",
+       "   </g>\n",
+       "  </g>\n",
+       " </g>\n",
+       " <defs>\n",
+       "  <clipPath id=\"pead59bdea8\">\n",
+       "   <rect height=\"217.44\" width=\"334.8\" x=\"24.2\" y=\"7.2\"/>\n",
+       "  </clipPath>\n",
+       " </defs>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "cases.diff().plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get a list of regions in Germany, we can use:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['Bayern',\n",
+       " 'Hamburg',\n",
+       " 'Mecklenburg-Vorpommern',\n",
+       " 'Niedersachsen',\n",
+       " 'Thüringen',\n",
+       " 'Saarland',\n",
+       " 'Baden-Württemberg',\n",
+       " 'Nordrhein-Westfalen',\n",
+       " 'Rheinland-Pfalz',\n",
+       " 'Sachsen',\n",
+       " 'Sachsen-Anhalt',\n",
+       " 'Hessen',\n",
+       " 'Schleswig-Holstein',\n",
+       " 'Berlin',\n",
+       " 'Bremen',\n",
+       " 'Brandenburg']"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "germany = ov.fetch_data_germany()\n",
+    "list(set(germany['Bundesland'].values))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get a list of Landkreise, we can use"
```

Comment:
This cell perhaps should be deleted. The cell below is the one showing the different ```Landkreise```

Reply:
The two cell are not just duplicates: 
![image](https://user-images.githubusercontent.com/1487169/89312122-29dca600-d677-11ea-94e7-34af98ce6efa.png)

What I'd offer is to remove the first unsorted list:
```python
list(set(germany['Bundesland'].values))
```

and leave the sorted one.

Reply:
You're right :+1:

Reply:
Thanks, well spotted. I have tidied this up as requested.